compiler/resolution/tuples.cpp

```diff
@@ -799,13 +800,14 @@ do_computeTupleWithIntent(bool valueOnly, IntentTag intent, Type* t)
 {
   INT_ASSERT(t->symbol->hasFlag(FLAG_TUPLE));
 
-  FnSymbol*  typeConstructor    = t->defaultTypeConstructor;
-  BlockStmt* instantiationPoint = typeConstructor->instantiationPoint;
-
   // Construct tuple that would be used for a particular argument intent.
 
   bool allSame = true;
   AggregateType* at = toAggregateType(t);
+
+  FnSymbol*  typeConstructor    = at->defaultTypeConstructor;
```

Comment:
I'm little leery of doing that since tuples are so mystical, but I'll give it a shot